app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int, **kwargs) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
```

Comment:
As it studied at Mate lectures, It's better to use just Person.people to add element to class attribute instead of using magic method __class__

Reply:
OK